### PR TITLE
pstoedit,plotutils: fix for darwin and mark not broken

### PR DESCRIPTION
**Pull request by jbaum98**
###### Motivation for this change

pstoedit and plotutils were marked broken for darwin, and pstoedit was not compiling due to a missing dependency on ApplicationServices.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/graphics/pstoedit/default.nix [hunk 1]
```diff
@@ -13,13 +13,14 @@ stdenv.mkDerivation rec {
...
+  buildInputs = [ zlib ghostscript imagemagick plotutils gd libjpeg libwebp libiconv ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
+    ApplicationServices
+  ]);
 
   meta = with stdenv.lib; {
     description = "Translates PostScript and PDF graphics into other vector formats";
     homepage = https://sourceforge.net/projects/pstoedit/;
     license = licenses.gpl2;
     maintainers = [ maintainers.marcweber ];
-    platforms = platforms.linux;
```
Comment (globin) [inline pkgs/tools/graphics/pstoedit/default.nix]: Don't remove this line, change it to `platforms.unix`

File: pkgs/tools/graphics/pstoedit/default.nix [hunk 2]
```diff
@@ -13,13 +13,14 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ zlib ghostscript imagemagick plotutils gd libjpeg libwebp ];
+  buildInputs = [ zlib ghostscript imagemagick plotutils gd libjpeg libwebp libiconv ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
```
Comment (globin) [inline pkgs/tools/graphics/pstoedit/default.nix]: I'd prefer having this line wrapped earlier

File: pkgs/tools/graphics/plotutils/default.nix [hunk 3]
```diff
@@ -52,6 +52,5 @@ stdenv.mkDerivation rec {
 
     license = stdenv.lib.licenses.gpl2Plus;
     maintainers = [ stdenv.lib.maintainers.marcweber ];
-    platforms = stdenv.lib.platforms.gnu;
```
Comment (globin) [inline pkgs/tools/graphics/plotutils/default.nix]: Don't remove this line, change it to `platforms.unix`

**Review by globin — CHANGES_REQUESTED** [review]

**Comment by jbaum98** [comment]
@globin thanks for the suggestions, I made those changes and pushed a new commit.

File: pkgs/tools/graphics/pstoedit/default.nix [hunk 4]
```diff
@@ -13,13 +13,16 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ zlib ghostscript imagemagick plotutils gd libjpeg libwebp ];
+  buildInputs = [ zlib ghostscript imagemagick plotutils gd libjpeg libwebp libiconv ]
```
Comment (LnL7) [inline pkgs/tools/graphics/pstoedit/default.nix]: libiconv is only needed on dariwn.

**Review by LnL7 — APPROVED** [review]

**Pull request merged**